card/db/migrate_core_cards/20180905061534_migrate_layouts.rb

```diff
@@ -0,0 +1,56 @@
+# -*- encoding : utf-8 -*-
+
+class MigrateLayouts < Card::Migration::Core
+  def up
+    Card.search type: "Layout" do |card|
+      update_layout_card card
+      create_head_rules card
```

Comment:
Layouts now start with body tag. The surrounding html stuff is added by `page_wrapper` in the html format. The head is defined by `*head` rules.